common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/ShapleyLoadOptions.java

```diff
@@ -0,0 +1,69 @@
+package ai.h2o.mojos.deploy.common.transform;
+
+/**
+ * Enum defining options for loading the mojo to enable Shapley predictions.
+ */
+public enum ShapleyLoadOptions {
```

Comment:
nit: Traditionally, enum is named after the representation of the single type and not the collective.